common-services-v1.yml

```diff
@@ -17,6 +17,7 @@ baseslave:
     build: ./slave
     volumes:
         - ./slave:/home/omero
+        - /ome:/ome
```

Comment:
No read-only for the purposes of this task (and I'll push to move to the bf container instead of slave)